### PR TITLE
Added multilingual support for webpage in English and Irish

### PR DESCRIPTION
**Pull request by kimaya1008**
Created a new script file to store language information

### CODE REVIEW

File: scripts/pageLngSupport.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        document.getElementById("textLang").innerHTML = "Roghnaigh teanga téacs: ";
+    }
+    else       
+    {
+        document.getElementById("title").innerHTML = "Spell Checker";
+        document.getElementById("webLang").innerHTML = "Select webpage language:  ";
+        document.getElementById("text").innerHTML = "Enter your text below: ";
+        document.getElementById("textLang").innerHTML = "Select text language: ";
+    }
+}
```
Comment (rjuliang) [inline scripts/pageLngSupport.js:19]: We might be able to create an object with this information for the final version

**Review by rjuliang — APPROVED** [review]
Hi Kimaya,
We may be able to make a few changes and add another language for the final version, however, I think this is so far a very good solution for the beta version.

File: scripts/pageLngSupport.js (hunk 1 above)
Comment (Pream9) [reply scripts/pageLngSupport.js:19]: ok

**Comment by Pream9** [comment]
If there is a chance to add the other language then we need to go with the Spanish language, I think.
As it is also the most speaking language in America.

**Pull request merged**